### PR TITLE
Add CFT scorecard integration

### PR DESCRIPTION
**Pull request by ocervell**
Integrates policy-library with CFT scorecard to run add ability to run audit on real CAI data (existing Forseti bucket or new bucket + dump)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,5 +49,10 @@ check_sample_files: ## Make sure each template in policies/templates has one sam
 check_format: ## Check that files have been formatted using opa fmt
 	@test $$(opa fmt -l lib/ validator/ | wc -l) -eq '0'
 
+.PHONY: audit
+audit:
+	@echo "Running config-validator audit ..."
+	@sh scripts/cft.sh -p ${PROJECT_ID} -o ${ORG_ID} -b ${CAI_BUCKET_NAME} -e ${EXPORT}
```
Comment (ocervell) [inline Makefile]: @morgante I've noticed that I can't use both flags `-p` and `-o` (if one is empty, then cft scorecard will hang), so not sure how to structure this call here

Comment (morgante) [reply Makefile]: I'd say your wrapper script should only pass through which value is non-empty.

Comment (ocervell) [reply Makefile]: Actually, scorecard doesn't support the `--organization` flag yet

File: scripts/cft.sh [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+setup_gcloud(){
+  if [ ! -z "$project_id" ]
+  then
+    gcloud config set core/project $project_id
+  fi
+}
+
+export_cai(){
+  if [[ $bucket_name == *"forseti"* ]]; then
+    echo "Forseti bucket detected. Renaming existing dumps to CFT scorecard format ..."
```
[the remark this answers is not in the document]
Comment (ocervell) [reply scripts/cft.sh]: Ok, I've left it here in the meantime, till it's implemented upstream.

File: scripts/cft.sh [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  if ! [ -f cft ]; then
+    echo "CFT Binary not found, downloading ..."
+    curl -o cft https://storage.googleapis.com/cft-scorecard/v0.2.0/${bin_name}
+    chmod +x cft
+  fi
+}
+
+run_cft(){
+  ./cft scorecard \
+  --policy-path=$policy_library_path \
```
[the remark this answers is not in the document]
Comment (ocervell) [reply scripts/cft.sh]: Breaks the script when I do for some reason ...

File: scripts/cft.sh [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  echo "  - Project: $project_id"
+  echo "  - Organization: $organization_id"
+  echo "  - Policy-library path: $policy_library_path"
+  echo "  - Do CAI export ? $export"
+}
+
+setup_gcloud(){
+  if [ ! -z "$project_id" ]
+  then
+    gcloud config set core/project $project_id
```
[the remark this answers is not in the document]
Comment (ocervell) [reply scripts/cft.sh]: Done.

**Comment by ocervell** [comment]
@morgante @AdrienWalkowiak small knits done; this is ready for another review

**Comment by ocervell** [comment]
@morgante @AdrienWalkowiak small knits done; this is ready for another review

**Pull request merged**